### PR TITLE
[11.0] [MIG] purchase_operating_unit

### PR DESCRIPTION
**Pull request by bjeficent**
Migration of purchase_operating_unit to v11

### CODE REVIEW

File: purchase_operating_unit/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+{
+    "name": "Operating Unit in Purchase Orders",
+    "summary": "An operating unit (OU) is an organizational entity part of a "
+               "company",
+    "version": "11.0.1.1.1",
+    "author": "Eficent, "
+              "Serpent Consulting Services Pvt. Ltd.,"
+              "Odoo Community Association (OCA)",
+    "website": "http://www.eficent.com",
```
Comment (nikul-serpentcs) [inline purchase_operating_unit/__manifest__.py]: @bjeficent URL Should be `https://github.com/OCA/operating-unit`

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

File: purchase_operating_unit/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# © 2015-17 Eficent Business and IT Consulting Services S.L.
+# - Jordi Ballester Alomar
+# © 2015-17 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
```
Comment (nikul-serpentcs) [inline purchase_operating_unit/__manifest__.py]: © -> Copyright and 2015-17 -> 2015-19

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Copyright

**Comment by bjeficent** [comment]
Comments attended, thanks for reviewing.

**Review by danypr92 — APPROVED** [review]

**Review by enricostano — APPROVED** [review]

**Comment by AaronHForgeFlow** [comment]
@nikul-serpentcs Could you please update your review bro?

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Comment by AaronHForgeFlow** [comment]
Oops I pushed from the wrong place. I will fix that.

**Comment by AaronHForgeFlow** [comment]
Github detects the commit hash is the same so it will not reconsider the user. I created a duplicate of this PR here #210

**Comment by AaronHForgeFlow** [comment]
:+1: Tested

**Pull request closed without merging**